pkgs/development/interpreters/php/default.nix

```diff
@@ -91,6 +91,12 @@ let
           configureFlags = [ "--enable-pcntl" ];
         };
 
+        # pcre functionality is tested in nixos/tests/php-pcre.nix
```

Comment:
It will be more noticeable to say at the top of the file that there is nixos php test available.